jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -1333,13 +1334,28 @@ public static DslViewResultsTree resultsTreeVisualizer() {
     return new DslViewResultsTree();
   }
 
+  /**
+   * Builds a ConstantThrouputTimer which paces samplers to achieve a target constant throughput.
+   *
+   * @param throughput specifies the target samples per minute.
+   * @param mode       specifies the calculation mode to apply.
+   * @return the timer for usage in test plan.
+   * @see DslConstantThroughputTimer
+   */
+
+  public static DslConstantThroughputTimer constantThroughputTimer(
```

Comment:
what do you think as naming this as throughputTimer? I think it might make api simpler if at any point we could need to implement support for throughput shaping timer. Eg:
```
throughputTimer(25) // for a fixed throughtput
throughputTimer() // for pre configured
  .rampTo(10, Duration.ofSeconds(5))
  .rampToAndHold(20, Duration.ofSeconds(5), Duration.ofSeconds(5))
  .rampToAndHold(30, Duration.ofSeconds(5), Duration.ofSeconds(5))
```

Additionally the JMeter documentation states: "N.B. although the Timer is called the Constant Throughput timer, the throughput value does not need to be constant.". So, maybe throughputTimer is more descriptive and avoid confusion, for example, if in the future we want to add support for somethig like `throughputTimer("${THROUGHPUT}")

Reply:
I suppose it depends if the goal is mirror JMeter i.e. call things the same for ease of use for people very familiar with JMeter or keep it as simple as possible. I was implementing this to mirror JMeter. Thoughts?

Reply:
Yes, that is a usual point of doubt: keep with names as similar as possible to JMeter ones as to ease migration to existing JMeter users, or change names to make them simpler (to reduce code) or more clear for users that are not used to JMeter. In this scenario the difference is quite small and I think benefits (flexibility and expressivness) added by the change are worth it over cons. I would expect jmeter users for the worst case scenario to search first by "constant..." and then by "throughtput" (which would show the element).

We are planning to add a JMeter users special section in guide, to list main differences in default values and names, to easily map from one to the other and understand what is being used by the DSL in each case, as to provide such users with less "surprises" and more control when using the DSL (eg: automatic inclusion of cookie & cache Manager).

Reply:
makes sense, i'll amend it